src/pages/index.js

```diff
@@ -87,18 +87,18 @@ const settings = {
 /*                            MODAL SUBMIT BUTTONS                            */
```

Comment:
From the 8th sprint all constants should be in file `constants.js` in folder `utils` to clean up the main file.